Tasks/AzureRmWebAppDeployment/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -1,7 +1,7 @@
 {
   "loc.friendlyName": "Azure App Service Deploy",
   "loc.helpMarkDown": "[More Information](https://aka.ms/azurermwebdeployreadme)",
-  "loc.description": "Update Azure Web App Services, Web App On Linux , Function Apps, Mobile Apps using Web Deploy / Kudu REST APIs",
+  "loc.description": "Update Azure WebApp Services On Windows, Web App On Linux with built-in images or docker containers, ASP.NET app, .NET Core app, PHP app, Function Apps, Mobile Apps, Api applications, Web Jobs using Web Deploy / Kudu REST APIs",
```

Comment:
Update Azure WebApp Services On Windows, Web App On Linux with built-in images or docker containers, ASP.NET,  .NET Core,  PHP, Python or Node based Web applications, Function Apps, Mobile Apps, Api applications, Web Jobs using Web Deploy / Kudu REST APIs